features/step_definitions/messaging_steps.rb

```diff
@@ -0,0 +1,52 @@
+When("I am on the inbox page") do
+  visit mailbox_inbox_path
+end
+  
+Then("I should see {string}") do |content|
+  expect(page).to have_content content
+end
+
+Given("I am logged in as {string}") do |name|
```

Comment:
Please separate the step definitions in two files. `basic_steps.rb` and `assertion_steps.rb`. Everything that is not an assertion should be in the `basic_steps.rb`